tools/python/convert_onnx_models_to_ort.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import glob
+import os
+import re
+import sys
+import tempfile
+
+import onnxruntime as ort
+
+
+def create_config_file(optimized_model_path, config_file_path):
+    script_path = os.path.dirname(os.path.realpath(__file__))
+    ci_build_py_path = os.path.abspath(os.path.join(script_path, '..', 'ci_build'))
+    sys.path.append(ci_build_py_path)
+
+    # create config file from all the optimized models
+    print("Creating configuration file for operators required by optimized models in {}".format(config_file_path))
+    from exclude_unused_ops import exclude_unused_ops  # tools/ci_build/exclude_unused_ops.py
+    exclude_unused_ops(optimized_model_path, config_path=None, ort_root=None, output_config_path=config_file_path)
+
+
+def convert(model_path: str):
+    models = glob.glob(os.path.join(model_path, '*.onnx'))
+
+    if len(models) == 0:
+        raise ValueError("No .onnx files were found in " + model_path)
+
+    # create temp directory to create optimized onnx format models in. currently we need this to create the
+    # config file with required operators. long term we could potentially do this from the ORT format model,
+    # however that requires a lot of infrastructure to be able to parse the flatbuffers schema for those files
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        for model in models:
+            model_filename = os.path.basename(model)
```

Comment:
Changes from original script:
  - input is directory containing multiple models instead of a single model
  - writes optimized onnx format model to temp dir so they are not user visible
  - calls create_config_file create configuration file from optimized onnx format models in temp dir